### DISCUSSION TITLE
Starter Game Sample - Compatible?

### DISCUSSION THREAD

**Discussion by VinnyVicious** [o3de/o3de#1885] [Q&A] [answered]
Is the Starter Game Sample for Lumberyard compatible with O3DE? 

https://docs.aws.amazon.com/lumberyard/latest/userguide/sample-level-starter-game.html

**Comment by amzn-leenguy** [comment]
I believe they are working on converting this. Still TBD on the date that it will be done. CC @amzn-melissaa

**Comment by amzn-melissaa** [comment, marked as answer]
There are certain systems that are not available yet in O3DE such as terrain and environmental effects. Some pieces of Starter Game can be brought over to O3DE right now such as the assets but also would require converting the materials over to work with the Atom renderer. As @amzn-leenguy pointed out, we are working on converting at least the asset gems from Starter Game so they work with Atom. Stay tuned!

**Reply by VinnyVicious** [reply]
Are you guys going to update this discussion once samples are available? Or is there a specific roadmap item to look for?

**Reply by amzn-melissaa** [reply]
We'll post here when they're available.

**Comment by VinnyVicious** [comment]
Has anything changed in regards to game samples?

**Reply by AMZN-Liv** [reply]
Hey @VinnyVicious - while we haven't migrated the Starter Game yet, we did release a sample game from our internal game jam last week. 
* Blog: https://aws.amazon.com/blogs/gametech/we-also-make-games-aws-first-internal-o3de-game-is-now-available/ 
* GitHub source: https://github.com/o3de/Planet_Storm 

We're continuing to work on getting new sample game content available - this is just the first.

**Comment by VinnyVicious** [comment]
Was this ever completed? Either the Lumberyard samples or the original CryEngine ones.

**Reply by guillaume-haerinck** [reply]
This will be progressively worked on via this repository, it is pretty much WIP but you can already open the map https://github.com/o3de/startergame-assets/pull/1

**Comment by harryclinton** [comment]
If you're testing compatibility, start with the base version first. Some mods like unlock all in[ carx street mod apk ](https://carxstreetmodi.com/unlock-all-cars-in-carx-street/)can affect stability. I usually compare performance before and after installing anything unofficial.

**Reply by harryclinton** [reply]
It should work fine with the starter sample, but double check your engine version and dependencies first. If you’re tweaking visuals later, this guide helped me: [https://lightroommodi.com/how-to-use-a-brush-in-adobe-lightroom/](https://lightroommodi.com/how-to-use-a-brush-in-adobe-lightroom/) for subtle adjustments.
